main.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-undef */
+
```

Comment:
- [ ] Kindly note that linters are very important they always help developers arrange their code and prevent future bugs. Kindly make sure you solve them instead of disabling them.